frontend/Nouns/CreateEditIntakeForm.tsx

```diff
@@ -342,6 +342,12 @@ export interface IntakeParagraphItem {
   textContent: string;
 }
 
+export interface IntakePageItem {
+  type: IntakeItemType.Page;
+  id: number;
+  intakeItems: IntakeItem[];
```

Comment:
This does not match the api response json that has `pageChildren` rather than `intakeItems`